docs/additional-functionality/delta-lake-support.md

```diff
@@ -67,6 +67,16 @@ that control the operation of the optimized write.
 | spark.databricks.delta.optimizeWrite.smallPartitionFactor   | 0.5     | Merge partitions smaller than this factor multiplied by the target partition size          |
 | spark.databricks.delta.optimizeWrite.mergedPartitionFactor  | 1.2     | Avoid combining partitions larger than this factor multiplied by the target partition size |
 
+Automatic compaction is supported only on Databricks platforms. The algorithm is similar but 
```

Comment:
Above this it says "Automatic compaction is not supported" which need to be updated.

Reply:
Ah, shoot. I thought I got that already. Let me read through it again.

Reply:
I've clarified the doc.